ws-server/src/server.rs

```diff
@@ -319,21 +319,20 @@ async fn background_task(
 	tokio::spawn(async move {
 		// Received messages from the WebSocket.
 		let mut rx_item = rx.next();
+		let mut submit_ping = Box::pin(tokio::time::sleep(ping_interval));
```

Comment:
can we use [tokio::IntervalStream](https://docs.rs/tokio-stream/latest/tokio_stream/wrappers/struct.IntervalStream.html) instead of having to these `Box::pin`?

`tokio::pin` is better because it pins it on stack.